recipes/qtconsole/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: qtconsole
+  version: {{ version }}
+
+source:
+  fn: qtconsole-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/q/qtconsole/qtconsole-{{ version }}.tar.gz
+  md5: c08ebebc7a60629ebadf685361ca0798
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - ipykernel >=4.1
+    - jupyter_client >=4.1
+    - jupyter_core
+    - pygments
+    - pyqt
+    - traitlets
+
+  run:
+    - python
+    - ipykernel >=4.1
+    - jupyter_client >=4.1
+    - jupyter_core
+    - pygments
+    - pyqt
+    - traitlets
+
+test:
+  commands:
+    - jupyter qtconsole --help                                              # [not linux]
+    - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'jupyter qtconsole --help'  # [linux]
+
+  imports:
+    - qtconsole
+    - qtconsole.tests
+
+about:
+  home: http://jupyter.org
+  license: BSD 3-Clause
+  summary: Jupyter Qt console
+
+extra:
+  recipe-maintainers:
+    - Carreau
```

Comment:
Added you, @Carreau. Thanks. 😄

When this gets merged, you will get an invite to join conda-forge. Once accepted, this will add you to the `qtconsole` team so that you have commit privileges on this feedstock/repo. Going forward there will be no additional action required on your part (beyond adding your GitHub handle to recipes like so). Please let me know if you have any questions and welcome to conda-forge.